src/runtime-rs/crates/hypervisor/src/device/driver/virtio_vsock.rs

```diff
@@ -118,15 +118,21 @@ const CID_RETRY_COUNT: u32 = 50;
 
 impl VsockDevice {
     pub async fn new(id: String) -> Result<Self> {
-        let (guest_cid, _vhost_fd) = generate_vhost_vsock_cid()
-            .await
-            .context("generate vhost vsock cid failed")?;
-
         Ok(Self {
             id,
-            config: VsockConfig { guest_cid },
+            config: VsockConfig {
+                guest_cid: libc::VMADDR_CID_ANY,
+            },
```

Comment:
IMO, the vsock config is first initialized in [prepare_vm_socket_config](https://github.com/kata-containers/kata-containers/blob/0f80dc636c3a4df6eb5cc3034e5702c87c17778e/src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs#L264), but it's not passed down as an argument in VsockDevice::new(). What's about changing the `VsockDevice::new(id)` to `VsockDevice::new(id, config) `and in the body is `{
    Self {id, config}
}`

Reply:
It sounds better. I will reflect your suggestion on the code. Thanks! 😉

Reply:
Done, I hope the change would meet your expectation. Thanks!